### ISSUE TITLE
aoc command line triggers rate limiter

### ISSUE THREAD

**Issue by dopplershift** [wimglenn/advent-of-code-data#132]
When using the `aoc` command line tool to run a bunch of solutions that aren't cached (new computer), I found it regularly hitting the internal rate limiter, triggering increasing exponential back-off (getting to > 80 seconds). Given that the library itself has control over this, it would be nice if the `aoc` tool could avoid this--maybe add a slight delay between running solutions or something?

**Comment by wimglenn** [comment]
The back-off _is_ adding a delay between running solutions. And I don't want any additional delays to the runner if we're not actually making network requests. 

I'm inclined to just say don't worry about this, since it only happens when the caches are cold.  80 seconds seems a bit excessive though, I'll probably put an upper bound on the exponential back-off in future.

**Comment by dopplershift** [comment]
Awesome, capping it at 10s seems perfectly fine. 

Just wanted to add that part of it was that I was getting messages telling me to "Slow down!" and my reaction was "I'm not in control here, you are!" 😆

**Comment by wimglenn** [comment]
That's a good point. I may reword it to be more informational that caches are cold, but it probably has to stay at WARNING level: an event at INFO or below would be invisible to the intended audience (people who may be using aocd in their CI and not preserving caches between runs)
